cmd/bedPeOverlap/bedPeOverlap.go

```diff
@@ -51,18 +90,20 @@ func bedpeOverlap(bedpeSelectFile string, bedpeInFile string, contactOutFile str
 
 	err = out.Close()
 	exception.PanicOnErr(err)
-
 }
 
 func usage() {
-	fmt.Print("bedpeOverlap - Returns all bedpe entries where both bed region pairs overlap a bedpe entry in the selectFile \n" +
-		"bedpe entries from in the inFile will be selected based on overlaps with the selectFile.\n" +
+	fmt.Print("bedpeOverlap - Filters bedpe entries based on overlaps from the select file.\n" +
+		"Default behavior expects a bedpe select file and returns entries where both ends of a bedpe entry from the input file" +
+		"overlap both ends of a bedpe entry from the select file.\n" +
 		"Usage:\n" +
-		"	bedpeOverlap [options] selectFile.bedpe inFile.bedPe out.bedpe\n\n")
+		"bedpeOverlap [options] selectFile inputFile.bedPe out.bedpe\n\n")
 	flag.PrintDefaults()
 }
 
 func main() {
+	var bedSelect *bool = flag.Bool("bedSelect", false, "Set select file to be a BED file instead of a bedpe.")
```

Comment:
Would it be appropriate to add that "Set select file to be a BED file instead of a bedpe. Returns entries where either end of a bedpe entry from the input file overlaps a bed entry from the select file"?